rabbitvcs/util/strings.py

```diff
@@ -0,0 +1,230 @@
+#
+# This is an extension to the Nautilus file manager to allow better
+# integration with the Subversion source control system.
+#
+# Copyright (C) 2006-2008 by Jason Field <jason@jasonfield.com>
+# Copyright (C) 2007-2008 by Bruce van der Kooij <brucevdkooij@gmail.com>
+# Copyright (C) 2008-2010 by Adam Plumb <adamplumb@gmail.com>
+#
+# RabbitVCS is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# RabbitVCS is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with RabbitVCS;  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+
+Additional strings support.
+
+"""
+
+import sys
+import codecs
+import re
+import six
+import locale
+
+__all__ = ["S", "IDENTITY_ENCODING", "UTF8_ENCODING", "SURROGATEESCAPE"]
```

Comment:
Any reason in particular you didn't put an underscore in here?  Having two Es next to each other like this is a little awkward.

Reply:
I presume you speak about `SURROGATEESCAPE`? The only reason I did not put an underscore is the standrd Python > 3.1 error handler is called `surrogateescape`. But I'm not stuck on it and if you prefer` SURROGATE_ESCAPE`, I can change it.